src/main/convert.js

```diff
@@ -58,12 +58,15 @@ export class ThriftFileConverter {
   constructor(
     thriftPath: string,
     transformName: string => string,
-    withsource: boolean
+    withsource: boolean,
+    excludeservice: boolean,
```

Comment:
[nit] naming should be pascalCase

Reply:
yeah i would like pascalCase as well, but i'm just following the flag standards that uber has set for this package